### DISCUSSION TITLE
production ready standard "post - like/dislike - comment" system

### DISCUSSION THREAD

**Discussion by tesla-cat** [fastapi/fastapi#8351] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
na
```

### Description

- many APPs have the same "post - like/dislike - comment" model, they only differ in contents
- can you provide a tutorial on building such a production ready system using FastAPI?

### Operating System

Windows

### Operating System Details

na

### FastAPI Version

na

### Python Version

na

### Additional Context

na

**Comment by chbndrhnns** [comment]
Maybe try fiverr.com or Upwork.com for this kind of work.
